src/creature.cpp

```diff
@@ -688,13 +692,13 @@ BlockType_t Creature::blockHit(Creature* attacker, CombatType_t combatType, int3
 	}
 
 	if (attacker) {
-		if (Player* attackerPlayer = attacker->getPlayer()) {
+		if (auto attackerPlayer = attacker->getPlayer()) {
```

Comment:
Could this be `const auto`
```suggestion
		if (const auto attackerPlayer = attacker->getPlayer()) {
```